### PR TITLE
#8301:  Applied more Portuguese translations

### LINKED ISSUE #8301: Apply more Portuguese translations
**Is your feature request related to a problem? Please describe.**
There are some phrases across the app that are not translated at all or are missing the Portuguese translations. 

**Describe the solution you'd like**
We should use currently used mechanism to provide additional translations. 

**Additional context**
The list of required strings to translate:
- "Edit Dashboard" to "Editar Painel"
- "Force refresh dashboard" to "Forçar actualização do painel"
- "Set auto-refresh interval" to "Definir intervalo de actualização automática"
- "Share Dashboard" to "Partilhar Painel"
- "Query timeout" to "Demora na Consulta"
- "visualization queries are set to timeout at ${action.timeout} seconds. " to "as consultas de visualização estão definidas para um tempo limite de ${action.timeout} segundos. "
- "Perhaps your data has grown, your database is under unusual load, or you are simply querying a data source that is too large to be processed within the timeout range. If that is the case, we recommend that you summarize your data further." to "Os dados poderão ter aumentado de volume, o banco de dados pode estar sob uma carga incomum ou a consulta de dados seja muito grande para ser processada dentro do intervalo de tempo limite. Se esse for o caso, recomendamos que resuma seus dados ainda mais."
- "Apply" to "Aplicar" to "Last day" to "Último dia"
- "Last week" to  "Último semana"
- "Last month" to "Último mês"
- "Last quarter" to "Último quarto"
- "Last year" to "Último ano"
- "No filter" to "Sem filtro"
- "Last" to "Último"
- "Next" to "Próximo"
- "defaults" to "padrões"
- "seconds" to "segundos"
- "minutes" to "minutos"
- "hours" to "horas"
- "days" to "dias"
- "weeks" to "semanas"
- "months" to "meses"
- "years" to "anos"
- "Start / end" to "Começar / fim"
- "Relative to today" to "Em relação a hoje"
- "Custom" to "Costume"
- "Time range" to "Intervalo de tempo"
- "No results found" to "Dados não encontrados"
- "Data" to "Dados"
- "Customize" to "Customizar"

### PR DESCRIPTION
**Pull request by oskarhinc**
### CATEGORY
Choose one
- [ ] Bug Fix
- [x] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY

Translated some phrases across the app to Portuguese.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API
- [x] Fixes #8301

### CODE REVIEW

**Comment by oskarhinc** [comment]
I'm closing and reopening this pull request to re-trigger Travis check as it seems to fail randomly

**Comment by oskarhinc** [comment]
@villebro do you know who can/should I mark as a reviewer of those/similiar changes?

**Comment by villebro** [comment]
This LGTM, but best to have someone with more frontend experience validate this; @etr2460 is this good to go?

**Comment by oskarhinc** [comment]
@mistercrunch I've seen you've merged a pull request related to Portuguese translations before, so could you review it?

**Review by SWoto — APPROVED** [review]
I applied the changes in my superset and it worked without any problems.

**Review by SWoto — APPROVED** [review]
I applied the changes in my superset and it worked without any problems.

**Review by mistercrunch — APPROVED** [review]
We have 2 Portuguese committers as @betodealmeida and @dpgaspar but happy to merge to move forward

File: superset/translations/pt/LC_MESSAGES/message.po [hunk 1]
```diff
@@ -4178,3 +4178,96 @@ msgstr "Alterado em"
...
+"are simply querying a data source that is too large to be processed "
+"within the timeout range. If that is the case, we recommend that you "
+"summarize your data further."
+msgstr ""
+"Os dados poderão ter aumentado de volume, "
+"o banco de dados pode estar sob uma carga incomum "
+"ou a consulta de dados seja muito grande para ser processada "
+"dentro do intervalo de tempo limite. Se esse for o caso, "
+"recomendamos que resuma seus dados ainda mais."
+
```
Comment (dpgaspar) [inline superset/translations/pt/LC_MESSAGES/message.po]: What do you think about, changing it to:
```
"O volume de dados poderá ter aumentado, "
"a base de dados pode estar sob uma carga incomum "
"ou a consulta ser demasiado grande para ser processada "
"dentro do intervalo de tempo limite. Se for esse o caso, "
"recomendamos que sumarize os dados ainda mais."
```

**Review by dpgaspar — CHANGES_REQUESTED** [review]

**Comment by amitmiran137** [comment]
closing due to inactivity

**Pull request closed without merging**